src/seedu/addressbook/AddressBook.java

```diff
@@ -256,7 +268,8 @@ private static void echoUserCommand(String userCommand) {
      *
      * @param args full program arguments passed to application main method
      */
-    private static void processProgramArgs(String[] args) {
+    @SuppressWarnings("unused")
```

Comment:
you can safely delete this method because it would still remain in the commit history.